recipe/patches/0001-loosen-requirements.patch

```diff
@@ -13,33 +13,26 @@ Co-authored-by: ngam
  1 file changed, 1 insertion(+), 4 deletions(-)
 
 diff --git a/tensorflow/tools/pip_package/setup.py b/tensorflow/tools/pip_package/setup.py
-index c3f29b7c92728..4719e3ce63b98 100644
+index face32bbb9a2e..6b251d8ac17ee 100644
 --- a/tensorflow/tools/pip_package/setup.py
 +++ b/tensorflow/tools/pip_package/setup.py
-@@ -88,7 +88,6 @@ def standard_or_nightly(standard, nightly):
-     'gast >= 0.2.1, <= 0.4.0',
-     'google_pasta >= 0.1.1',
-     'h5py >= 2.9.0',
+@@ -92,7 +92,6 @@ def standard_or_nightly(standard, nightly):
+     # TODO(b/239052279): replace with external dependency on JAX repo once JAX
+     # no longer relies on TF.
+     'jax >= 0.3.15',
```

Comment:
Do we need to add jax to the requirement?

Reply:
Done.